#!/bin/sh /usr/share/dpatch/dpatch-run
## 07-manpage-symlink.dpatch by Sebastian Rasmussen <sebras@hotmail.com>
##
## DP: Adjusts manpage to match source-code (Closes: #453824).

@DPATCH@

diff -Naur thttpd-2.25b.orig/thttpd.8 thttpd-2.25b/thttpd.8
--- thttpd-2.25b.orig/thttpd.8	2005-06-29 17:50:56.000000000 +0000
+++ thttpd-2.25b/thttpd.8	2007-12-27 13:57:11.000000000 +0000
@@ -90,7 +90,7 @@
 Note, though, that if you are using the chroot option, the symlink
 checking is unnecessary and is turned off, so the safe way to save
 those CPU cycles is to use chroot.
-The config-file option names for this flag are "symlinkcheck" and "nosymlinkcheck".
+The config-file option names for this flag are "symlinks" and "nosymlinks".
 .TP
 .B -v
 Do el-cheapo virtual hosting.
